src/azure-cli/azure/cli/command_modules/appconfig/_params.py

```diff
@@ -74,48 +75,50 @@ def load_arguments(self, _):
+        c.argument('prefix', help="This prefix will be appended to the front of imported keys. Not applicable for feature flags.")
+        c.argument('source', options_list=['--source', '-s'], choices=['file', 'appconfig', 'appservice'], validator=validate_import, help="The source of importing. Note that importing feature flags from appservice is not supported.")
         c.argument('yes', help="Do not prompt for preview.")
+        c.argument('skip_features', help="Import only key values and exclude all feature flags. By default, all feature flags will be imported from file or appconfig. Not applicable for appservice.", arg_type=get_three_state_flag())
 
     with self.argument_context('appconfig kv import', arg_group='File') as c:
         c.argument('path', help='Local configuration file path. Required for file arguments.')
-        c.argument('format_', options_list=['--format'], choices=['json', 'yaml', 'properties'], help='Imported file format. Required for file arguments.')
-        c.argument('depth', validator=validate_import_depth, help="Depth for flatterning the json or yaml file to key-value paris. Flattern to the deepest level by default.")
+        c.argument('format_', options_list=['--format'], choices=['json', 'yaml', 'properties'], help='Imported file format. Required for file arguments. Currently, feature flags are only supported in json format.')
+        c.argument('depth', validator=validate_import_depth, help="Depth for flattening the json or yaml file to key-value pairs. Flatten to the deepest level by default. Not appicable for property files or feature flags.")
         # bypass cli allowed values limition
-        c.argument('separator', validator=validate_separator, help="Delimiter for flatterning the json or yaml file to key-value pairs. Required for importing hierarchical structure. Not applicable for property files. Supported values: '.', ',', ';', '-', '_', '__', '/', ':' ")
+        c.argument('separator', validator=validate_separator, help="Delimiter for flattening the json or yaml file to key-value pairs. Required for importing hierarchical structure. Not applicable for property files or feature flags. Supported values: '.', ',', ';', '-', '_', '__', '/', ':' ")
 
     with self.argument_context('appconfig kv import', arg_group='AppConfig') as c:
         c.argument('src_name', help='The name of the source App Configuration.')
-        c.argument('src_connection_string', validator=validate_connection_string, help="Connections of access key and endpoint of the source store. ")
-        c.argument('src_key', help='If no key specified, import all keys by default. Support star sign as filters, for instance abc* means keys with abc as prefix. Similarly, *abc and *abc* are also supported.')
+        c.argument('src_connection_string', validator=validate_connection_string, help="Combination of access key and endpoint of the source store. ")
```

Comment:
Is there a way to generate connection string quickly?

Reply:
For this arg_group, we need 2 connection strings to connect to two different AppConfig stores. While the first (default) connection string can be easily configured, the second is not that straight forward as the second AppConfig store may or may not be in the same subscription that the user is logged in to. Moreover, since we only use this connection string for import/export commands when  arg_group='AppConfig', implementing a solution to set this quickly will be an overkill. 

---
In reply to: [355225819](https://github.com/Azure/azure-cli/pull/11467#discussion_r355225819) [](ancestors = 355225819)